src/System.Net.Http.WinHttpHandler/tests/project.lock.json

```diff
@@ -173,10 +173,10 @@
           "lib/dotnet/System.IO.FileSystem.Primitives.dll": {}
         }
       },
-      "System.Linq/4.0.0-beta-22816": {
+      "System.Linq/4.0.0-beta-22703": {
         "dependencies": {
-          "System.Collections": "4.0.10-beta-22816",
-          "System.Runtime": "4.0.20-beta-22816"
+          "System.Collections": "4.0.0-beta-22703",
+          "System.Runtime": "4.0.0-beta-22703"
```

Comment:
I pulled master, and ran "build.cmd /p:LockDependencies=true". I didn't actually change anything myself, just let the system run through the process.  I'm going to investigate a bit more and see why it might have gone backward...